NextExample/next.config.js

```diff
@@ -1,14 +1,25 @@
 const { withExpo } = require('@expo/next-adapter');
 
+const withPlugins = require('next-compose-plugins');
+
+const withBundleAnalyzer = require('@next/bundle-analyzer')({
+  enabled: process.env.ANALYZE === '1',
+});
+
+const disableMinification = process.env.DISABLE_MINIFICATION === '1';
+
 /** @type {import('next').NextConfig} */
-module.exports = withExpo({
+module.exports = withPlugins([withBundleAnalyzer, withExpo], {
   // transpilePackages is a Next.js +13.1 feature.
   // older versions can use next-transpile-modules
   eslint: {
     ignoreDuringBuilds: true,
   },
   transpilePackages: ['react-native-reanimated', 'react-native', 'expo'],
   webpack(config, _options) {
+    if (disableMinification) {
+      config.optimization.minimizer = [];
+    }
```

Comment:
I think we should add to documentation information what does mean DISABLE_MINIFICATION and ANALYZE, and how to use it.